### ISSUE TITLE
library not installing on OSX

### ISSUE THREAD

**Issue by sh3d2** [ulwanski/unix-dgram-socket#1]
is there any possibility this would install on OSX?

**Comment by ulwanski** [comment]
It should, any errors or logs you can share?

**Comment by sh3d2** [comment]
```
/tmp npm i unix-dgram-socket --save

npm WARN enoent ENOENT: no such file or directory, open '/private/tmp/package.json'
npm WARN tmp No description
npm WARN tmp No repository field.
npm WARN tmp No README data
npm WARN tmp No license field.

npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for unix-dgram-socket@1.0.6: wanted {"os":"linux","arch":"any"} (current: {"os":"darwin","arch":"x64"})
npm ERR! notsup Valid OS:    linux
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   darwin
npm ERR! notsup Actual Arch: x64
```

**Comment by ulwanski** [comment]
It's restricted to linux only because package was not tested on OS X, I will perform necessary tests and  release fixed version within 3 days.

**Comment by ulwanski** [comment]
Problem was solved in release 1.0.7
